### PR TITLE
Tensor Processing Unit, Peter Incorvaja

### PR DESCRIPTION
**Pull request by wavejumpr**

### CODE REVIEW

File: terms/tensorprocessingunit_pi.bib [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+{
+    paun2013determinism,
+    title={On the determinism of multi-core processors},
+    author={Paun, Vladimir-Alexandru and Monsuez, Bruno and Baufreton, Philippe},
+    booktitle={French Singaporean Workshop on Formal Methods and Applications},
+    pages={35,36},
+    year={2013}
+}
+
+@misc
```
Comment (jp-um) [inline terms/tensorprocessingunit_pi.bib]: Not a scholarly reference: find book or research article

File: terms/tensorprocessingunit_pi.bib [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    sato,
+    author = {Sato, Kaz and Young, Cliff and Patterson, David},
+    title  = {An in-depth look at Google's first Tensor Processing Unit (TPU)},
+    howpublished ={Website},
+    year={2017},
+    month=5,
+    url ={https://cloud.google.com/blog/products/gcp/an-in-depth-look-at-googles-first-tensor-processing-unit-tpu}
+}
+
+@misc
```
Comment (jp-um) [inline terms/tensorprocessingunit_pi.bib]: Not a scholarly reference: find book or research article

File: terms/tensorprocessingunit_pi.bib [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    google,
+    author = {Google},
+    title  = {XLA - TensorFlow, compiled},
+    howpublished ={Website},
+    year={2017},
+    month=3,
+    url    = {https://developers.googleblog.com/2017/03/xla-tensorflow-compiled.html}
+}
+
+@misc
```
Comment (jp-um) [inline terms/tensorprocessingunit_pi.bib]: Not a scholarly reference: find book or research article

Comment (wavejumpr) [reply terms/tensorprocessingunit_pi.bib]: I'm aware that it is not, but the only scholarly references i could find specifically on TPUs is the one published. All other material is either Google's documentation on https://cloud.google.com/tpu/ or blogposts.

**Comment by wavejumpr** [comment]
I have referenced that in fact

**Comment by wavejumpr** [comment]
since it's a proprietary architecture, the only information one could find is from Google, where they only issued that particular paper, and of course the documentation on https://cloud.google.com/tpu/ and Google Developer Blogposts.

**Comment by jp-um** [comment]
U el hallihom mela... will have a read through and let you know.

*Dr Jean-Paul Ebejer**,*  BSc (Hons) (Melita), MSc (Imperial), DPhil (Oxon.)

*Lecturer*
Centre for Molecular Medicine and BioBanking,
Office 320, Biomedical Sciences Building,
University of Malta, Msida MSD 2080.
T: (00356) 2340 3263

<https://twitter.com/malteseunderdog>

**Comment by jp-um** [comment]
Merged, thanks.

**Pull request merged**